ci/pull-upstreams.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
what's different now that this file needs to be committed?

Reply:
This file can pull more than one other repo into one PR while the other only is able to pull one.